packages/phishing-controller/src/PhishingController.ts

```diff
@@ -45,6 +46,20 @@ export interface EthPhishingDetectConfig {
   version: number;
 }
 
+/**
+ * @type PhishingDetectState
+ *
+ * Interface defining expected type of the stalelist.json file. Also matches the state object.
+ * @property allowlist - List of approved origins (legacy naming "whitelist")
+ * @property blocklist - List of unapproved origins (legacy naming "blacklist")
+ * @property fuzzylist - List of fuzzy-matched unapproved origins
+ * @property tolerance - Fuzzy match tolerance level
+ * @property lastUpdated - Timestamp of last update.
+ */
+export interface PhishingDetectState extends EthPhishingDetectConfig {
```

Comment:
I'm not sure we should be extending from `EthPhishingDetectConfig` here, because that type essentially represents the constructor arguments for this class. If we add other arguments in the future that have nothing to do with the phishing detect state, they'd be added to that type.

Maybe we could inline each of those properties here instead of extending?

Reply:
Also, I'm not sure about this type name. I guess it's fair that this is "phishing detect state", given that it's in the state. But it's used in a few other places as a representation of the phishing list configuration, not just the state.

So maybe we could call it `PhishingStalelist`? Or something along those lines.